src/components/views/ScreeningsCard.vue

```diff
@@ -0,0 +1,104 @@
+<script>
+import { defineComponent } from 'vue';
+import MoviePoster from '@/components/common/Movie/MoviePoster.vue';
+import MovieLength from '@/components/common/Movie/MovieLength.vue';
+import AppButton from '@/components/common/App/AppButton.vue';
+import MovieTitle from '@/components/common/Movie/MovieTitle.vue';
+import AppTag from '@/components/common/App/AppTag.vue';
+import { getScreeningsByMovieAndDate } from '@/api/service/Screenings';
+
+export default defineComponent({
+  components: { MoviePoster, MovieLength, AppButton, MovieTitle, AppTag },
+  props : {
+    movieData: {
+      type: Object,
+      required: true,
+    },
+    daySelected: {
+      type: Date,
+      default: new Date().toISOString().slice(0, 10)
+    },
+    titleSelected: {
+      type: String,
+      default: 'All movies',
+    }
+  },
+  data() {
+    return {
+      screenings: [],
+    }
+  },
+  watch: {
+    daySelected() {
+      this.getScreenings()
+    },
+  },
+  methods: {
+    getScreeningTime(screening) {
+      return screening.datetime.substring(11, 16)
+    },
+    async getScreenings() {
+      try {
+        const response = await getScreeningsByMovieAndDate(this.movieData.id, this.daySelected);
```

Comment:
Also - we could get the screenings information for all the movies on the top level and only pass the data to each of the cards - otherwise we do *a ton* of requests to the API which not only makes our app behave slower (because of so much network communication) but also makes each of the cards load on their own time - which might look weird in off cases.

Ideally you can do a single request to `/seances` once and properly parse the data you have along with the information about the movies.
Getting this data for a *specific* movie is what we will do on the single movie page 😄